src/test/java/uk/gov/pay/api/service/SearchRefundsServiceTest.java

```diff
@@ -85,18 +82,20 @@ public void getAllRefundsShouldReturnCorrectTotalAndPageAndDisplaySize() {
                 .assertThat("$.results[0].charge_id", is(extChargeId))
                 .assertThat("$.results[0].amount_submitted", is(98))
                 .assertThat("$.results[0].status", is("available"))
-                .assertThat("$.results[0].links", not(hasKey("refunds")))
-                .assertThat("$.results[0].links.self.href", is(format("http://publicapi.test.localhost/v1/payments/%s/refunds/%s", extChargeId, refundId1)))
-                .assertThat("$.results[0].links.payment.href", is(format("http://publicapi.test.localhost/v1/payments/%s", extChargeId)))
+                .assertThat("$.results[0]", not(hasKey("links")))
+                .assertThat("$.results[0]._links", not(hasKey("refunds")))
```

Comment:
same here, not sure this should change, because it's testing the response from the connector pact, right?

Reply:
actually this is the external response from public api, so it should change